### PR TITLE
Verify email

### PR DESCRIPTION
**Pull request by mohammad-rework**

### CODE REVIEW

File: src/components/elements/query-image.tsx [hunk 1]
```diff
@@ -8,6 +8,7 @@ type QueryImageProps = {
     data: ImageDataLike
     height?: string
     width?: string
+    max_width?: string
```
Comment (sean-binary) [inline src/components/elements/query-image.tsx:11]: Do we still need this?

Comment (mohammad-rework) [reply src/components/elements/query-image.tsx:11]: `max_width` is still using in other components like src/components/layout/nav.js

**Review by sean-binary — APPROVED** [review]
LGTM 🎉

**Review by sean-binary — APPROVED** [review]

File: src/pages/academy/subscription/index.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+                    <LinkButton secondary="true" to="/academy/">
+                        {localize('Take me to Academy')}
+                    </LinkButton>
+                </Flex>
+            </SectionContainer>
+        </>
+    )
+}
+
+export default Subscription
```
Comment (sean-binary) [inline src/pages/academy/subscription/index.tsx:100]:
```suggestion
export default WithIntl()(Subscription)
```
We also will need this for future page translations.

**Review by sean-binary — APPROVED** [review]

**Review by sean-binary — APPROVED** [review]

**Pull request merged**